android/quest/src/main/assets/configs/app/application_config.json

```
@@ -39,6 +39,7 @@
       "Flag"
     ]
   },
+  "logQuestionnaireLocation": true,
```

This was updated to an `enum LocationLogOptions`

```suggestion
  "logGpsLocation": [
     "QUESTIONNAIRE"
  ],
```